### ISSUE TITLE
Feature Request: better testing support

### ISSUE THREAD

**Issue by jademcgough** [socialpandas/sidekiq-superworker#40]
I'd love it if superworkers responded to sidekiq testing methods (see https://github.com/mperham/sidekiq/wiki/Testing#testing-worker-queueing-fake) and delegated them to sidekiq workers as appropriate, like `Superworker.drain` draining workers sequentially.

Willing to take a stab at this later if this seems like a good idea.

**Comment by tombenner** [comment]
Agreed, this'd be great to have. Feel free to take a stab at it, if you like!
